ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -65,7 +66,13 @@ export class ImportCommand extends IronfishCommand {
   async importFile(path: string): Promise<AccountImport> {
     const resolved = this.sdk.fileSystem.resolve(path)
     const data = await this.sdk.fileSystem.readFile(resolved)
-    return JSONUtils.parse<AccountImport>(data)
+    try {
+      const decodedWords = bech32m.bech32m.decode(data).words
+      const decodedBytes = bech32m.bech32m.fromWords(decodedWords)
+      return JSONUtils.parse<AccountImport>(decodedBytes.toString())
+    } catch (e) {
+      return JSONUtils.parse<AccountImport>(data)
+    }
```

Comment:
```suggestion
    try {
      const decodedWords = bech32m.bech32m.decode(data).words
      const decodedBytes = bech32m.bech32m.fromWords(decodedWords)
      data = decodedBytes.toString()
    } catch (e DecodingErrorWords DecodingErrorBytes) {
      // failure suggests already in json format
    }
    return JSONUtils.parse<AccountImport>(data)
```

Reply:
Not sure of exact error types, but this captures the essence

Reply:
`bech32` seems to only throw `Error` with no types, unless I'm missing something: https://github.com/bitcoinjs/bech32/blob/master/src/index.ts#L111